components/basecomponents/StoryText.tsx

```diff
@@ -49,14 +51,6 @@ const StoryText = ({ languageText, languagePlay, audio, id, onPlaying }: StoryTe
     return scrollToRef(phraseRef, containerRef);
   }, [phraseRef?.current?.offsetTop]);
 
-  const playPhrase = (start: number) => {
-    if (audio !== null) {
-      audio.currentTime = start;
-      audio.play();
-      onPlaying(true);
-    }
-  };
-
   const playing = (phrase: StoryPhrase) => {
     type ObjectKey = keyof typeof phrase;
```

Comment:
These `ObjectKey` hacks seem unnecesary. Consider just accessing the right value directly:

```
    const start = languagePlay === 'uk' ? phrase.start_uk : phrase.start_cs;
    const end = languagePlay === "uk" ? phrase.end_uk : phrase.end_cs;
    return audio?.currentTime > start && audio?.currentTime < end;
```


 The long term solution is to refactor StoryPhrase to match the `Phrase` interface, but that's a larger separate task :)

Reply:
I would leave it for this situation and we can create refactoring ticket, where will be describe more issue there. Because hacks are in more cases, there.